support-frontend/assets/helpers/productCatalogToProductOption.ts

```diff
@@ -1,47 +1,20 @@
-// describes options relating to a product itself - only relevant for paper currently
 import type {
-	ActiveProductKey,
-	ActiveRatePlanKey,
-} from 'helpers/productCatalog';
-
-const NoProductOptions = 'NoProductOptions';
-const Saturday = 'Saturday';
-const SaturdayPlus = 'SaturdayPlus';
-const Sunday = 'Sunday';
-const SundayPlus = 'SundayPlus';
-const Weekend = 'Weekend';
-const WeekendPlus = 'WeekendPlus';
-const Sixday = 'Sixday';
-const SixdayPlus = 'SixdayPlus';
-const Everyday = 'Everyday';
-const EverydayPlus = 'EverydayPlus';
-const NewspaperArchive = 'NewspaperArchive';
-
-export type ProductOptions =
-	| typeof NoProductOptions
-	| typeof Saturday
-	| typeof SaturdayPlus
-	| typeof Sunday
-	| typeof SundayPlus
-	| typeof Weekend
-	| typeof WeekendPlus
-	| typeof Sixday
-	| typeof SixdayPlus
-	| typeof Everyday
-	| typeof EverydayPlus
-	| typeof NewspaperArchive;
-
-export type PaperProductOptions =
-	| typeof Saturday
-	| typeof Sunday
-	| typeof Weekend
-	| typeof Sixday
-	| typeof Everyday
-	| typeof SaturdayPlus
-	| typeof SundayPlus
-	| typeof WeekendPlus
-	| typeof SixdayPlus
-	| typeof EverydayPlus;
+	PaperProductOptions,
```

Comment:
This file contains functions which were in the productOptions file but which are specific to support-frontend so I've left them there for now